src/plugins/job_submit/pbs/spank_pbs.c

```diff
@@ -65,14 +65,28 @@ int slurm_spank_task_init(spank_t sp, int ac, char **av)
 		spank_setenv(sp, "PBS_ARRAY_ID", val, 1);
 	if (spank_getenv(sp, "SLURM_ARRAY_TASK_ID", val, sizeof(val)) ==
 	    ESPANK_SUCCESS)
-		spank_setenv(sp, "PBS_ARRAY_INDEX", val, 1);
+		spank_setenv(sp, "PBS_ARRAYID", val, 1);
```

Comment:
same remark as with talamini: if the slurm vars exists during profile, you can also set those there

Reply:
Yes, but that means maintaining it here.

Reply:
it means maintaining some bash code/script, not set of c patches (and you can move the talamini code here as well; so even less work)

Reply:
you build these rpms anyway, so not sure why you would need a different repo for 1 or 2 files. but "same difference" i guess

Reply:
I need the talamini repo for generating the pbs node file, which I am not going to bring into the main slurm repo.